src/settings.rs

```diff
@@ -350,20 +350,17 @@ impl Store for FileSystemStore {
             ),
         }
 
-        match self.toml[TILE_SIZE].as_integer() {
-            Some(tile_size) => {
-                let tile_size = tile_size as i32;
-                if crate::engine::AVAILABLE_TILE_SIZES.contains(&tile_size) {
-                    settings.tile_size = tile_size;
-                } else {
-                    log::error!("Settings: unsupported `{}`: {}", TILE_SIZE, tile_size);
-                    log::info!(
-                        "Available tile sizes: {:?}",
-                        crate::engine::AVAILABLE_TILE_SIZES
-                    );
-                }
+        if let Some(tile_size) = self.toml[TILE_SIZE].as_integer() {
+            let tile_size = tile_size as i32;
+            if crate::engine::AVAILABLE_TILE_SIZES.contains(&tile_size) {
+                settings.tile_size = tile_size;
+            } else {
+                log::error!("Settings: unsupported `{}`: {}", TILE_SIZE, tile_size);
+                log::info!(
+                    "Available tile sizes: {:?}",
+                    crate::engine::AVAILABLE_TILE_SIZES
+                );
```

Comment:
I don't have a strong preference between `if let` and `match` here, but this change removes the `log::error!("Settings: missing `{}` entry.", TILE_SIZE)` and I want that to stay here.